code/modules/clothing/spacesuits/rig/suits/alien.dm

```diff
@@ -152,3 +152,39 @@
 
 	initial_modules = list(
 		)
+//ChompEdit Begins
+
+/obj/item/weapon/rig/vox/engineering
```

Comment:
You are missing `req_access = list(access_engine)` if you want to have this locked to engineering only